modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js

```diff
@@ -986,6 +1006,13 @@
 			} else if ( pageCanLoadEditor ) {
 				// Allow instant switching to edit mode, without refresh
 				$caVeEdit.off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
+				// FANDOM - UGC-3932 experiment start
+				if ( isUserInExperiment() ) {
+					// Preload VisualEditor scripts
+					$caEdit.on('mouseover.ve-target-source', this.preloadVeScripts.bind(this));
+					$caVeEdit.on('mouseover.ve-target', this.preloadVeScripts.bind(this));
+				}
```

Comment:
So we load faster VE only when user is on article and click edit button? 

If user enters edit mode direct by url, everything will load normaly?

Reply:
Yes, there isn't change when user enters editor directly by url as it still needs to download page content and resources (in most times from cache)